docs/src/assets/examples/array.raml

```diff
@@ -0,0 +1,58 @@
+#%RAML 1.0
+#baseUri: http://www.tshirt.com/api
```

Comment:
Is it possible to slim down some of this? We shouldn't need the examples committed to the repo, and we might be able to cut a few other things out as well.